client/src/components/overview/overview-components/Expanded.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react"
+import styled from 'styled-components';
+
+const StyledExpandedView = styled.div`
+    min-width:60rem;
+    min-height: 40rem;
+    display: flex;
+    margin: auto;
+    text-align: center;
```

Comment:
I believe this line is unnecessary. When using flex display you align-items to center or justify-content. See https://marina-ferreira.github.io/tutorials/css/flexbox/